pontoon/base/views.py

```diff
@@ -177,9 +177,20 @@ def authors_and_time_range(request, locale, slug, part):
     paths = [part] if part != 'all-resources' else None
 
     translations = Translation.for_locale_project_paths(locale, project, paths)
-
+    filters = Q(entity__resource__project=project)
```

Comment:
You [also](https://github.com/mozilla/pontoon/blob/master/pontoon/base/models.py#L2434) need to filter by locale and filter out obsolete entities:
`filters = Q(entity__resource__project=project) & Q(entity__obsolete=False) & Q(locale=locale)`

Reply:
ah, thanks, ill update